docs/es/manuals/addressing.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Direccionamiento en Defold
+brief: Este manual explica cómo Defold ha solucionado los problemas de direccionamiento
+
+---
+
+# Direccionamiento
+
+En un juego, el código que creamos tiene que ser capaz de encontrar cada objeto y componente, con tal de mover, escalar, animar, eliminar y manipular aquello que el jugador percibe del juego. En Defold, el direccionamiento es el mecanismo que se encarga de que ello sea posible.
+
+## Identificadores
+
+Defold usa direcciones (o URL, cómo se mostrará más adelante) para referenciar objetos y componentes del juego. Estas direcciones se basan en identificadores, tal y como se muestra en los siguientes ejemplos.
+
+'''lua
+local id = factory.create("#enemy_factory")
+label.set_text("my_gameobject#my_label", "Hello World!")
+
+local pos = go.get_position("my_gameobject")
+go.set_position(pos, "/level/stuff/other_gameobject")
+
+msg.post("#", "hello_there")
+local id = go.get_id(".")
+'''
+
+Empecemos con un ejemplo muy simple. Suponiendo que tenemos un objeto simple con un sprite y un script como componentes, se vería de la siguiente forma:
+
+![bean in editor](images/addressing/bean_editor.png)
+
+Ahora vamos a deshabilitar el sprite cuando el juego inicia, para hacerlo aparecer cuando lo necesitemos. Esto se hace fácilmente con el siguiente fragmento de código en "controller.script":
+
+'''lua
+function init(self)
+msg.post("#body", "disable") -- <1&gt;
+end
+'''
+1. No te preocupes si no acabas de entender el '#'. Más tarde se explicará.
+
+Esto hará que cuando el juego inicie, el script "direccione" el sprite por el identificador "body" y use esa dirección para mandarle un "mensaje" con "disable". Este mensaje hará que el componente sprite se esconda a la vista. La siguiente imagen ayuda a comprender su funcionamiento.
+
+![bean](images/addressing/bean.png)
+
+El identificador es un parámetro sobre el que nosotros decidimos el valor. En este caso, se le da al objeto el identificador de "bean", al igual que al script se le nombra "controller" y al sprite "body"
+
+::: Anotación
```

Comment:
This line should not be translate. It is used to highlight certain text in the generated documentation.

```
::: sidenote
```